vaadin-quarkus/vaadin-quarkus-extension-deployment/pom.xml

```diff
@@ -0,0 +1,66 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+    <modelVersion>4.0.0</modelVersion>
+    <parent>
+        <groupId>com.vaadin</groupId>
+        <artifactId>vaadin-quarkus-extension-parent</artifactId>
+        <version>24.8-SNAPSHOT</version>
+        <relativePath>../pom.xml</relativePath>
+    </parent>
+
+    <artifactId>vaadin-quarkus-extension-deployment</artifactId>
+    <name>Vaadin Quarkus Extension Deployment</name>
+    <description>Deployment dependency for a Quarkus project.</description>
+
+    <repositories>
+        <!-- The order of definitions matters. Explicitly defining central here to make sure it has the highest priority. -->
+
+        <!-- Main Maven repository -->
+        <repository>
+            <id>central</id>
+            <url>https://repo.maven.apache.org/maven2</url>
+            <snapshots>
+                <enabled>false</enabled>
+            </snapshots>
+        </repository>
+        <repository>
+            <id>vaadin-prereleases</id>
+            <url>https://maven.vaadin.com/vaadin-prereleases</url>
+            <snapshots>
+                <enabled>true</enabled>
+            </snapshots>
+        </repository>
+    </repositories>
+
+    <dependencies>
+        <dependency>
+            <groupId>com.vaadin</groupId>
+            <artifactId>vaadin-quarkus-extension</artifactId>
+            <version>${project.version}</version>
+        </dependency>
+        <dependency>
+            <groupId>com.vaadin</groupId>
+            <artifactId>vaadin-quarkus-deployment</artifactId>
+            <version>${vaadin.quarkus.version}</version>
```

Comment:
I guess you dont this `${vaadin.quarkus.version}` here, as there is a dependencyManagement in the parent, vaadin-quarkus-extension-parent, right?

Reply:
Good catch! Let me fix it

Reply:
yeah, i saw that also. how often do we update that version?

Reply:
> yeah, i saw that also. how often do we update that version?

Not so frequently. Usually when having a new minor for `vaadin-quarkus`, so that we pin the latest Quarkus LTS version.

Reply:
The property on the platform POM file can be updated if needed when we make the vaadin-quarkus update PR